Dockerfile

```diff
@@ -9,13 +9,10 @@ RUN apk add --no-cache bash
 USER ${TURING_USER}
 
 ARG TURING_UI_DIST_PATH=ui/build
-ARG SWAGGER_UI_DIST_PATH=api/api/swagger-ui-dist
 
 ENV TURINGUICONFIG_SERVINGDIRECTORY "./turing-ui"
-ENV OPENAPICONFIG_SWAGGERUICONFIG_SERVINGDIRECTORY "./swagger-ui"
```

Comment:
This variable is used by [spf13/viper](https://github.com/spf13/viper) to set a corresponding config value:
https://github.com/gojek/turing/blob/b603d60763fd73d7b0104d5e88d1402b15d0e468/api/turing/config/config.go#L340

If the config value is not passed, then turing API wouldn't serve Swagger UI:

https://github.com/gojek/turing/blob/b603d60763fd73d7b0104d5e88d1402b15d0e468/api/turing/server/application.go#L127

Reply:
It is set in the _helper.tpl file.